install_sct

```diff
@@ -242,11 +242,11 @@ if uname -a | grep -i  darwin > /dev/null 2>&1; then
 
   # Fix for non-English Unicode systems on MAC
   if [ -z ${LC_ALL} ]; then
-    echo "export LC_ALL=en_US.UTF-8" >> ~/.bash_profile
+    export LC_ALL=en_US.UTF-8
```

Comment:
not going to work with (t)csh.

Reply:
Can it enter this if statement in Linux system? (If not, then disregard my comment about csh)

Reply:
We have an `IF` statement to check, we execute only on macOS:
`if uname -a | grep -i  darwin > /dev/null 2>&1; then`

Reply:
sorry the previous lines were hidden by github. All good then 👍

Reply:
@jcohenadad plz approve

Reply:
@abtahizadeh as mentioned in https://github.com/neuropoly/spinalcordtoolbox/pull/1881#discussion_r197626836, this PR needs to be tested before merging. You mentioned you had issues with pip that you will be fixing.